src/components/AppHeader/AppHeader.tsx

```diff
@@ -64,85 +112,125 @@ const AppHeader: React.FC<AppHeaderProps> = ({ pathname, allAgencyNames }) => {
      * When on desktop only, the dropdown nav links should close if any of the other ones
      * are still open. This next set of logic handles that.
      */
-    // if (
-    //   index === 0 &&
-    //   isOpen[1] === true &&
-    //   width > USWDS_BREAKPOINTS.DESKTOP
-    // ) {
-    //   setIsOpen([isOpen[0], false]);
-    // } else if (
-    //   index === 1 &&
-    //   isOpen[0] === true &&
-    //   width > USWDS_BREAKPOINTS.DESKTOP
-    // ) {
-    //   setIsOpen([false, isOpen[1]]);
-    // }
+    if (width > USWDS_BREAKPOINTS.DESKTOP) {
```

Comment:
This feels awfully fiddly and like there's gotta be a way to do it with a loop or something.

Reply:
Totally agreed!